_posts/2022-03-02-getting-started-project-flotta.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title:  "Getting started with flotta operator 1"
+date:   2022-23-02 11:44:17 -0400
+categories: flotta
+author: Ondra Machacek
+tags:
+  - guide
+  - flotta
+  
+summary-1: Getting started guide with flotta
+---
+In this blog post series we would like to show you how to get started with the flotta project.
+The flotta operator is the Kubernetes operator used to manage the workloads of the edge devices via
+Kubernetes API. And so to manage the edgedevices and workloads you need to have Kubernetes or OpenShift up and
+running. Unfortunatelly the flotta operator is not yet published on the operator hub, so we have to deploy it manually from the github repo.
```

Comment:
I wonder if it makes sense to publish it, because installing it from the published `operator.yaml` is way more simpler. But maybe producing another blog, when that method is available makes sense as well.

Reply:
I'd suggest to publish the image and release the `operator.yaml` with a tag v0.0.1 (so the user will know how to match it to the version of the device later-on) and update this blog to use that method.
It will reduce the need of the user to install build tools(git, make), golang, d/w dependencies and more. better experience overall.

Reply:
Ok, will do.